### PR TITLE
Conn 1278-Removed unused files and completed development.

### PR DESCRIPTION
**Pull request by sailajaa**

### CODE REVIEW

File: CONNECT-Webservices [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 9b70635d9deca083dc2141f24826d6d5f1e5b22b
```
Comment (christophermay07) [inline CONNECT-Webservices:1]: This and Common-Types should be dependencies, not subprojects.

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/pom.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            <groupId>org.connectopensource</groupId>            
+            <artifactId>AuditRepositoryCore</artifactId>            
+            <version>${project.parent.version}</version>        
+        </dependency>        
+        <dependency>            
+            <groupId>org.connectopensource</groupId>            
+            <artifactId>CORERule2_2_0Webervices</artifactId>            
+            <version>4.0.9-SNAPSHOT</version>        
+        </dependency>
+<<<<<<< HEAD
```
Comment (christophermay07) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/pom.xml]: Unresolved merge conflict here

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/pom.xml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    </parent>    
+    <artifactId>CORE_X12DocSubmissionAdapter10</artifactId>    
+    <packaging>war</packaging>
+    
+    <name>CONNECT Adapter - ${project.artifactId}</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.connectopensource</groupId>            
+            <artifactId>AuditRepositoryCore</artifactId>
```
Comment (christophermay07) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/pom.xml]: Audit Repo code should not be implemented yet, so this dependency should be removed.

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterX12DocSubmissionImpl.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+ *  DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ *  (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ *  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ *  ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ *  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ *  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package gov.hhs.fha.nhinc.corex12.docsubmission.realtime._10.adapter;
+
+<<<<<<< HEAD
```
Comment (christophermay07) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterX12DocSubmissionImpl.java]: Unresolved merge conflict (and several more below)

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterX12DocSubmissionSecured.java [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+ *  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package gov.hhs.fha.nhinc.corex12.docsubmission.realtime._10.adapter;
+
+import gov.hhs.fha.nhinc.common.nhinccommonadapter.AdapterBatchSubmissionAckResponseType;
+import gov.hhs.fha.nhinc.common.nhinccommonadapter.AdapterBatchSubmissionResponseSecuredType;
+import gov.hhs.fha.nhinc.common.nhinccommonadapter.AdapterBatchSubmissionSecuredRequestType;
+import gov.hhs.fha.nhinc.common.nhinccommonadapter.AdapterCOREEnvelopeRealTimeResponseType;
+import gov.hhs.fha.nhinc.common.nhinccommonadapter.AdapterCOREEnvelopeRealTimeSecuredRequestType;
+<<<<<<< HEAD
```
Comment (christophermay07) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterX12DocSubmissionSecured.java]: Unresolved merge conflict (and several more below)

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterX12DocSubmissionUnsecured.java [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+ *  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package gov.hhs.fha.nhinc.corex12.docsubmission.realtime._10.adapter;
+
+import gov.hhs.fha.nhinc.common.nhinccommonadapter.AdapterBatchSubmissionAckResponseType;
+import gov.hhs.fha.nhinc.common.nhinccommonadapter.AdapterBatchSubmissionRequestType;
+import gov.hhs.fha.nhinc.common.nhinccommonadapter.AdapterBatchSubmissionResponseType;
+import gov.hhs.fha.nhinc.common.nhinccommonadapter.AdapterCOREEnvelopeRealTimeRequestType;
+import gov.hhs.fha.nhinc.common.nhinccommonadapter.AdapterCOREEnvelopeRealTimeResponseType;
+<<<<<<< HEAD
```
Comment (christophermay07) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterX12DocSubmissionUnsecured.java]: Unresolved merge conflict (and several more below)

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/webapp/WEB-INF/cxf-servlet.xml [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+        </cxf:features>
+    </cxf:bus>
+
+    <jaxws:endpoint xmlns:tns="urn:gov:hhs:fha:nhinc:adaptercoresecured" id="AdapterDocSubmissionCoreSecured"
+        address="/CORETransaction/AdapterCORETransactionSecured" serviceName="tns:AdapterCORETransactionSecured" endpointName="tns:AdapterCORETransactionSecured_Port"
+        implementor="#AdapterCORESecured" wsdlLocation="classpath:wsdl/AdapterCORERule2.2.0Secured.wsdl">
+        <jaxws:properties>
+            <entry key="ws-security.signature.properties" value="file:///${nhinc.properties.dir}/signature.properties" />
+            <entry key="ws-security.encryption.properties" value="file:///${nhinc.properties.dir}/truststore.properties" />
+            <entry key="mtom-enabled" value="true" />
```
Comment (christophermay07) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/webapp/WEB-INF/cxf-servlet.xml]: MTOM should not be enabled -- change for all references in this file.

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterX12DocSubmissionImpl.java [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+import gov.hhs.fha.nhinc.messaging.server.BaseService;
+
+=======
+>>>>>>> c94ee3ebb63cf1b1ca9ad92cce83646df79c44a4
+/**
+ *
+ * @author sadusumilli
+ */
+<<<<<<< HEAD
+public class AdapterX12DocSubmissionImpl extends BaseService {
```
Comment (christophermay07) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterX12DocSubmissionImpl.java]: All new class names for CORE X12 Document Submission (RealTime) should be "**CORE_X12DSRealTime**" -- e.g., AdapterCORE_X12DSRealTimeImpl for this class.

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeImpl.java [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+import org.apache.log4j.Logger;
+import gov.hhs.fha.nhinc.messaging.server.BaseService;
+
+/**
+ *
+ * @author sadusumilli
+ */
+public class AdapterCORE_X12DSRealTimeImpl extends BaseService {
+
+    private static final Logger LOG = Logger.getLogger(AdapterCORE_X12DSRealTimeImpl.class);
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeImpl.java]: unused logger.

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeSecured.java [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+    }
+
+    @Override
+    public AdapterBatchSubmissionAckResponseType batchSubmitTransactionDeferredRequestSecured(AdapterBatchSubmissionSecuredRequestType abssrt) {
+        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+    }
+
+    @Override
+    public AdapterCOREEnvelopeRealTimeResponseType realTimeTransactionSecured(AdapterCOREEnvelopeRealTimeSecuredRequestType a) {
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeSecured.java]: use a better variable name than "a", should be descriptive.  Also, you're not passing the context to the impl class.

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeSecured.java [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+
+/**
+ *
+ * @author sadusumilli
+ */
+@BindingType(value = javax.xml.ws.soap.SOAPBinding.SOAP12HTTP_BINDING)
+@Addressing(enabled = true)
+public class AdapterCORE_X12DSRealTimeSecured implements gov.hhs.fha.nhinc.adaptercoresecured.AdapterCORETransactionSecuredPortType {
+
+    private WebServiceContext context;
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeSecured.java]: needs to be declared as an @resource

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeSecured.java [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+    }
+
+    @Override
+    public AdapterCOREEnvelopeRealTimeResponseType realTimeTransactionSecured(AdapterCOREEnvelopeRealTimeSecuredRequestType a) {
+
+        return new AdapterCORE_X12DSRealTimeImpl(outboundCOREX12DSRealTime).realTimeTransactionSecured(a);
+    }
+
+    public void setContext(WebServiceContext context) {
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeSecured.java]: remove this method

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeUnsecured.java [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+
+/**
+ *
+ * @author sadusumilli
+ */
+@BindingType(value = javax.xml.ws.soap.SOAPBinding.SOAP12HTTP_BINDING)
+@Addressing(enabled = true)
+public class AdapterCORE_X12DSRealTimeUnsecured implements gov.hhs.fha.nhinc.adaptercore.AdapterCORETransactionPortType {
+
+    private WebServiceContext context;
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeUnsecured.java]: same comments as the secured impl.

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/webapp/WEB-INF/cxf-servlet.xml [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+        </cxf:features>
+    </cxf:bus>
+
+    <jaxws:endpoint xmlns:tns="urn:gov:hhs:fha:nhinc:adaptercoresecured" id="AdapterDocSubmissionCoreSecured"
+        address="/CORETransaction/AdapterCORETransactionSecured" serviceName="tns:AdapterCORETransactionSecured" endpointName="tns:AdapterCORETransactionSecured_Port"
+        implementor="#AdapterCORESecured" wsdlLocation="classpath:wsdl/AdapterCORERule2.2.0Secured.wsdl">
+        <jaxws:properties>
+            <entry key="ws-security.signature.properties" value="file:///${nhinc.properties.dir}/signature.properties" />
+            <entry key="ws-security.encryption.properties" value="file:///${nhinc.properties.dir}/truststore.properties" />            
+            <entry key="ws-security.timestamp.validator">
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/webapp/WEB-INF/cxf-servlet.xml]: timestamp validator isn't necessary, only for large payload streaming.

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeImpl.java [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+    private OutboundCORE_X12DSRealTime outboundCOREX12DSRealTime;
+
+    /*
+     *
+     */
+    public AdapterCORE_X12DSRealTimeImpl(OutboundCORE_X12DSRealTime outboundCOREX12DSRealTime) {
+        this.outboundCOREX12DSRealTime = outboundCOREX12DSRealTime;
+    }
+
+    public AdapterCOREEnvelopeRealTimeResponseType realTimeTransactionSecured(AdapterCOREEnvelopeRealTimeSecuredRequestType a) {
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeImpl.java]: There should only be one method here.  Called by both secured and unsecured.  Should take context as input and convert to assertiontype.  This should then call the orchimpl for the adapter from the X12 core service.

File: Product/Production/Adapters/pom.xml [hunk 16]
```diff
@@ -16,7 +17,8 @@
         <module>DocumentSubmission_a0</module>
         <module>PatientDiscovery_a0</module>
         <!--module>Direct</module-->
-    </modules>
+    <module>CORE_X12DocSubmissionAdapter</module>
```
Comment (christophermay07) [inline Product/Production/Adapters/pom.xml]: This should be CORE_X12DocSubmission_a0

File: Product/Production/Adapters/CORE_X12DocSubmissionAdapter/pom.xml [hunk 17]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>Adapters</artifactId>
+        <groupId>org.connectopensource</groupId>
+        <version>4.4.0-SNAPSHOT</version>
+    </parent>    
+    <artifactId>CORE_X12DocSubmissionAdapter10</artifactId>
```
Comment (christophermay07) [inline Product/Production/Adapters/CORE_X12DocSubmissionAdapter/pom.xml]: This should be CORE_X12DocSubmission_a0

**Pull request closed without merging**